Userland/Demos/CatDog/CatDog.h

```diff
@@ -23,19 +23,25 @@ class CatDog final : public GUI::Widget {
     virtual void context_menu_event(GUI::ContextMenuEvent& event) override;
 
     void track_cursor_globally();
+    void set_skin(String const& skin_name, String const& skin_path)
+    {
+        m_skin_name = skin_name;
+        m_skin = *Gfx::Bitmap::load_from_file(skin_path);
+    };
     void start_the_timer() { m_timer.start(); }
 
     Function<void()> on_click;
     Function<void(GUI::ContextMenuEvent&)> on_context_menu_request;
 
+    String get_skin_name() { return m_skin_name; }
```

Comment:
Please don't prefix getters with `get_` as per the [code style guidelines](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md):
 
>  Precede setters with the word "set". Use bare words for getters. Setter and getter names should match the names of the variables being set/gotten.

Also, no need for a string copy.

```suggestion
    String const& skin_name() { return m_skin_name; }
```